### PR TITLE
[GC@Init] React Rails Gem

### PR DESCRIPTION
**Pull request by pinifloyd**
- https://github.com/reactjs/react-rails

Include React into Rails project. Seems like we can use it without pain with jQuery too, but not sure - we should check it.

/cc @ErrorSoul @rkichan

### CODE REVIEW

**Comment by ErrorSoul** [comment]
why we can't use cdn? 
I thought that react is a single library or like angular with many libraries?

File: config/environments/development.rb [hunk 1]
```diff
@@ -59,4 +59,7 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  # React js builds: https://github.com/reactjs/react-rails
+  config.react.variant = :development
```
Comment (ErrorSoul) [inline config/environments/development.rb:64]: what doing this setting?

Comment (pinifloyd) [inline config/environments/development.rb:64]: Setting up environment I guess :) See https://github.com/reactjs/react-rails#reactjs-builds

**Comment by pinifloyd** [comment]
We can use CDN - it's not a problem, but if we'll use CDN we'll lost powerful helpers for rails as React it's just a library (that true, you right) but rails-react is nice gem with many cool features.

File: config/environments/development.rb (hunk 1 above)
Comment (ErrorSoul) [inline config/environments/development.rb:64]: :+1:

**Comment by ErrorSoul** [comment]
This is for my understanding) ok - merge

**Comment by pinifloyd** [comment]
Sure, no problem :) BTW: rails-react gem is official gem from react team, if I mind true. Will merge it later, want take a look into docs one more time.

**Pull request merged**